integrations/tensorflow/iree_tf_compiler/TFL/LowerGlobalTensors.cpp

```diff
@@ -14,6 +14,10 @@
 #include "mlir/Support/LLVM.h"
 #include "mlir/Transforms/DialectConversion.h"
 #include "tensorflow/compiler/mlir/lite/ir/tfl_ops.h"
+#include "third_party/llvm/llvm-project/mlir/include/mlir/Dialect/Tosa/IR/TosaOps.h"
+#include "third_party/llvm/llvm-project/mlir/include/mlir/IR/BuiltinAttributes.h"
+#include "third_party/llvm/llvm-project/mlir/include/mlir/IR/BuiltinOps.h"
+#include "third_party/tensorflow/compiler/mlir/lite/ir/tfl_ops.h"
```

Comment:
You've got leftover google3 paths here from includes getting auto-added :-)

Reply:
Should be fixed up. I really hate the autoinclude.